04_numeronym/SandhyaSMani/04_numeronym.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+	fmt.Fprintln(out, numeronyms("accessibility", "Kubernetes", "abc"))
+}
+
+func numeronyms(vals ...string) []string {
+	var numeronymsArray = []string{}
+	for _, val := range vals {
+		var lengthVal = len(val)
```

Comment:
I'd just call it `s` or `slice` instead of `numeronymsArray`
and `l` or `length` instead of `lengthVal` - it's up to you though, it is a bit of a question of taste, however shorter variable names are [recommended](https://github.com/golang/go/wiki/CodeReviewComments#variable-names) in go.